tests/test_dataset.py

```diff
@@ -167,7 +167,7 @@ def get_multichanneled_data(dtype: type) -> np.ndarray:
 def test_create_wk_dataset_with_layer_and_mag() -> None:
     delete_dir("./testoutput/wk_dataset")
 
-    ds = WKDataset.create("./testoutput/wk_dataset", scale=(1, 1, 1))
+    ds = WKDataset.create(Path("./testoutput/wk_dataset"), scale=(1, 1, 1))
```

Comment:
The `./` is probably redundant (?)

Reply:
Also, it might make sense to store "testoutput" and "testdata" in constants in this file, rather than include it in so many strings. e.g. `TEST_OUTPUT_DIR = Path("testoutput")` and then down here `create(TEST_OUTPUT_DIR / "wk_dataset", scale=…)`

Reply:
Or more radically, @jstriebel do you think we should try to avoid all `/` occurrences in strings and rather always use pathlibs `/` operator?

Reply:
I think so, yes, to make this (more) Windows-compatible. This could simply be
```suggestion
    ds = WKDataset.create(Path("testoutput", "wk_dataset"), scale=(1, 1, 1))
```

or (my personal preference)
```suggestion
    ds = WKDataset.create(Path("testoutput") / "wk_dataset", scale=(1, 1, 1))
```